lesson_07/homeworks_07.py

```diff
@@ -31,27 +33,56 @@ def multiplication_table(number):
 # task 2
 """  Написати функцію, яка обчислює суму двох чисел.
 """
-
+def sumVal(a, b):
```

Comment:
```suggestion
def sum_val(a, b):
```
Правила ПЕП8 нада поважати